### PR TITLE
Fork support for skein Clients when grpcio>=1.18.0

### PR DESCRIPTION
**Pull request by nateagr**
Hello,

Here is the PR following the discussion here: https://github.com/jcrist/skein/issues/167

### CODE REVIEW

File: skein/utils.py [hunk 1]
```diff
@@ -82,12 +84,15 @@ def grpc_fork_support_disabled():
     gRPC (>= 1.15) results in extraneous error logs currently. For now we
     explicitly disable fork support for gRPC clients we create.
     """
-    key = 'GRPC_ENABLE_FORK_SUPPORT'
-    try:
-        os.environ[key] = '0'
-        yield
-    finally:
-        del os.environ[key]
+    if version.parse(grpcio.__version__) < version.parse('1.18.0'):
```
Comment (jcrist) [inline skein/utils.py]: I'd use `distutils.version.LooseVersion` instead, as it's part of the standard library:

```
if LooseVersion(grpcio.__version__) < '1.18.0':
    ...
```

Comment (nateagr) [reply skein/utils.py]: I agree 👍

File: skein/utils.py [hunk 2]
```diff
@@ -82,12 +84,15 @@ def grpc_fork_support_disabled():
     gRPC (>= 1.15) results in extraneous error logs currently. For now we
     explicitly disable fork support for gRPC clients we create.
     """
-    key = 'GRPC_ENABLE_FORK_SUPPORT'
-    try:
-        os.environ[key] = '0'
+    if LooseVersion(pkg_resources.get_distribution("grpcio").version) < '1.18.0':
```
Comment (jcrist) [inline skein/utils.py]: `pkg_resources` is also not standard library. Instead it looks like `grpc` version may be found at:

```python
try:
    from grpc import __version__ as GRPC_VERSION
except AttributeError:
    from grpc._grpcio_metadata import __version__ as GRPC_VERSION
```

from https://github.com/grpc/grpc/pull/17580.

**Comment by nateagr** [comment]
Hello @jcrist ,
Some tests seem to be flaky no ?

**Comment by jcrist** [comment]
Only on Travis (I haven't been able to debug why yet, sometimes the docker setup just fails). Anyway, this PR is good, thanks! Merging.

**Pull request merged**